packages/smart-wallet/package.json

```diff
@@ -33,6 +33,7 @@
     "@agoric/notifier": "^0.5.0",
     "@agoric/store": "^0.8.0",
     "@agoric/vat-data": "^0.4.0",
+    "@agoric/vats": "^0.11.0",
```

Comment:
vats depends on smart-wallet. Can we avoid the circular dep? Perhaps by passing a kit-maker in privateArgs? Is there any package `makeMyAddressNameAdminKit` could live in for re-use?

Reply:
Agreed it's the least bad option. I don't think moving the package has risk for the release because the release is using built assets and if anything goes wrong in the package refactoring they just won't build. But it will take time we don't want to spend. Incidentally, `internal` won't work because that's only for modules with no sdk dependencies and it depends on `@agoric/store`. Of course that may be in Endo eventually.